python/cudf/cudf/core/column/column.py

```diff
@@ -385,25 +385,47 @@ def copy(self, deep=True):
                 children=self.base_children,
             )
 
-    def view(self, newcls, **kwargs):
-        """View the underlying column data differently using a subclass of
-        ColumnBase
+    def view(self, dtype):
+        """
+        View the data underlying a column as different dtype.
```

Comment:
Add documentation of the requirements and limitations of this method.

Reply:
Done